R/pred_se_wrap.R

```diff
@@ -32,7 +33,8 @@
 pred_se_wrap <- function(df_trans, var_interest, model,
                          type = 'levels', base_rn = 1,
                          at_var_interest = NULL,
-                         vcov_mat = NULL){
+                         vcov_mat = NULL,
+                         dof = model$df.residual){
```

Comment:
Let's change `pred_se_wrap` to not have default arguments for `vcov_mat` or `dof`, since it's not designed to be called independently by the user.